dll/win32/shell32/wine/brsfolder.c

```diff
@@ -518,7 +531,12 @@ static void BrsFolder_CheckValidSelection( browse_info *info, LPTV_ITEMDATA lptv
     else
         EnableWindow(GetDlgItem(info->hWnd, IDC_BROWSE_FOR_FOLDER_NEW_FOLDER), TRUE);
 
+#ifdef __REACTOS__
+    if (bEnabled)
+            SendMessageW(info->hWnd, BFFM_ENABLEOK, 0, bEnabled);
+#else
     SendMessageW(info->hWnd, BFFM_ENABLEOK, 0, bEnabled);
+#endif
```

Comment:
```suggestion
```

Reply:
@HBelusca  I have implemented the CALLBACK BrowserCallbackProc that enable the OK button when CheckBestDriver function return TRUE, by finding the best driver to the current device.
So far everything works fine.
After that, the state of the button is overwritten by the call to BrsFolder_CheckValidSelection function.
Supose that's the solution.